recipes/xlsxio/all/conanfile.py

```diff
@@ -104,20 +113,37 @@ def package_info(self):
         self.cpp_info.set_property("cmake_module_file_name", "xlsxio")
         self.cpp_info.set_property("pkg_config_name", "xlsxio")
 
+        ziplib = "libzip::libzip" if self.options.with_libzip else "minizip-ng::minizip-ng"
```

Comment:
I think the logic here should be something like this? Because if not using minizip-ng you will be adding the wrong componen anyway:

```
        ziplib = "minizip::minizip"
        if self.options.with_libzip:
            ziplib = "libzip::libzip"
        elif self.options.get_safe("with_minizip_ng"):
            ziplib = "minizip-ng::minizip-ng"
```

Reply:
Probably the fail in ci is related to that

Reply:
@czoido 
Sorry for my late response.
Thanks a lot!
It works fine on my environment.
Try to pass CI.